plugins/CMakeLists.txt

```diff
@@ -49,7 +49,7 @@ if(OBS_CMAKE_VERSION VERSION_GREATER_EQUAL 3.0.0)
   add_obs_plugin(mac-syphon PLATFORMS MACOS)
   add_obs_plugin(mac-videotoolbox PLATFORMS MACOS)
   add_obs_plugin(mac-virtualcam PLATFORMS MACOS)
-
+  add_obs_plugin(nv-filters PLATFORMS WINDOWS)
```

Comment:
```suggestion
  add_obs_plugin(nv-filters PLATFORMS WINDOWS)
  
```
Re-add blank line before `check_obs_browser()`.

Reply:
done

Reply:
I still don't see it. This group of lines should be:
```
  add_obs_plugin(mac-virtualcam PLATFORMS MACOS)
  add_obs_plugin(nv-filters PLATFORMS WINDOWS)

  check_obs_browser()
```

Reply:
oh i had misread; fixed